tools/db_bench_tool.cc

```diff
@@ -4476,8 +4489,21 @@ class Benchmark {
         FLAGS_blob_compaction_readahead_size;
     options.blob_file_starting_level = FLAGS_blob_file_starting_level;
 
-    if (FLAGS_cache_size > 0) {
-      options.blob_cache = cache_;
+    if (FLAGS_use_blob_cache) {
+      if (FLAGS_use_shared_block_and_blob_cache) {
+        options.blob_cache = cache_;
+      } else {
+        if (FLAGS_blob_cache_size <= 0) {
+          fprintf(stderr,
+                  "Unable to create a standalone blob cache if blob_cache_size "
+                  "<= 0.\n");
+          exit(1);
+        }
+        LRUCacheOptions co;
```

Comment:
Maybe we should a function for this if we plan to support different caches: `lru`, `fastlru`, `clock` and also secondary cache.

Reply:
We don't have to do this now. Since clock cache has some bugs, and fastlru is mainly for sst blocks.